semgrep/event-with-info-msg.yaml

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) HashiCorp, Inc.
```

Comment:
Can this be placed in a hidden directory, ie `.semgrep`?

Reply:
Done!